packages/multimedia/media-driver/package.mk

```diff
@@ -0,0 +1,15 @@
+# SPDX-License-Identifier: GPL-2.0
+# Copyright (C) 2019-present Team LibreELEC (https://libreelec.tv)
+
+PKG_NAME="media-driver"
+PKG_VERSION="82c0e6f7f6fd920218031a1d202b4ae06e3cf148"
+PKG_SHA256="6f689d174a7a9597bc1d4b368530f822f29c2a26200c44bead6d885754b0e4b8"
+PKG_ARCH="x86_64"
+PKG_LICENSE="MIT"
+PKG_SITE="https://01.org/linuxmedia"
+PKG_URL="https://github.com/intel/media-driver/archive/$PKG_VERSION.tar.gz"
+PKG_DEPENDS_TARGET="toolchain libva libdrm gmmlib"
+PKG_LONGDESC="media-driver: The Intel(R) Media Driver for VAAPI is a new VA-API (Video Acceleration API) user mode driver supporting hardware accelerated decoding, encoding, and video post processing for GEN based graphics hardware."
+PKG_TOOLCHAIN="cmake"
```

Comment:
needed if only cmake is at the package ?

Reply:
removed